backend.py

```diff
@@ -340,6 +341,7 @@ def __init__(self, board, robots):
         self.present_deck = self.create_card_pack()
         self.past_deck = set()
         self.game_round = 1
+        self.game_over = False
```

Comment:
S timhle atributem se zde uz dal nepracuje. Je tedy nutny?

Reply:
já vůbec nechápu jakto že mi to stáhlo soubory a změny z toho winneru a z tama to zmizelo :-o